Mathlib/GroupTheory/SpecificGroups/KleinFour.lean

```diff
@@ -64,20 +64,18 @@ instance : IsKleinFour (DihedralGroup 2) where
   card_four := by simp only [Nat.card_eq_fintype_card]; rfl
   exponent_two := by simp [DihedralGroup.exponent]
 
-instance {G : Type*} [Group G] [IsKleinFour G] :
-    IsAddKleinFour (Additive G) where
+instance {G : Type*} [Group G] [IsKleinFour G] : IsAddKleinFour (Additive G) where
   card_four := by rw [← IsKleinFour.card_four (G := G)]; congr!
   exponent_two := by simp
 
-instance {G : Type*} [AddGroup G] [IsAddKleinFour G] :
-    IsKleinFour (Multiplicative G) where
+instance {G : Type*} [AddGroup G] [IsAddKleinFour G] : IsKleinFour (Multiplicative G) where
   card_four := by rw [← IsAddKleinFour.card_four (G := G)]; congr!
   exponent_two := by simp
 
 namespace IsKleinFour
 
 @[to_additive]
-instance instFinite {G : Type*} [Group G] [IsKleinFour G] : Finite G :=
+scoped instance instFinite {G : Type*} [Group G] [IsKleinFour G] : Finite G :=
```

Comment:
If the performance is actually affected, we should add a comment explaining why this is scoped

Reply:
Even if it is not affected there's no harm in adding a comment anyway!

Reply:
If it is not affected I'm not sure that we should be scoping this instance.